content/contribute.md

```diff
@@ -46,7 +46,9 @@ minor modifications.
 ## Sharing Your Tools
 
 If you would like us to feature an API or tool that you’ve developed for use
-with our products and services, please send us an email at support@maxmind.com
+with our products and services, please <a
+href="https://support.maxmind.com/hc/en-us/requests/new">contact our support
```

Comment:
```suggestion
href="https://support.maxmind.com/hc/en-us/requests/new"> contact our support
```
<img width="771" alt="Screenshot 2024-10-09 at 3 37 21 PM" src="https://github.com/user-attachments/assets/1bf9750c-1138-4202-89fa-c1721172a999">

Reply:
D'oh, need to use markdown links, not html https://github.com/maxmind/dev-site/pull/1199/commits/0b966ce68993b78f6ced13c97d79f9ee344b97f6